server/go.mod

```diff
@@ -3,6 +3,7 @@ module go.etcd.io/etcd/server/v3
 go 1.16
 
 require (
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
```

Comment:
Can you run `go mod tidy` on this?

Reply:
Already done before commit. Maybe i need to set some specific --compat?

Reply:
Please try calling `./scripts/fix.sh`. 

It should go over all the modules.

Reply:
@ptabor done.